include/sensors/core/mmr920C04.hpp

```diff
@@ -265,7 +265,7 @@ struct __attribute__((packed, __may_alias__)) Temperature {
     [[nodiscard]] static auto to_temperature(uint32_t reg) -> sq15_16 {
         float temperature =
             CONVERT_TO_CELSIUS * (static_cast<float>(reg) / MAX_SIZE);
-        return convert_to_fixed_point(temperature, 15);
+        return convert_to_fixed_point(temperature, 16);
```

Comment:
why don't we just make a `static constexpr S15Q16 = 16` in the same header that provides `convert_to_fixed_point` or something so we can use that instead of writing the number everywhere

Reply:
yep @fsinapi just suggested the same thing.